src/server/extensions/commands/builtins/wikipedia/style.css

```
@@ -18,6 +18,17 @@
   background: var(--border-color);
 }
 
+.wiki-thumb--logo {
+  display: block;    display: block;
```

Stylelint (17.12.0)

[error] Duplicate property "display" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)